{"tests": [{"data": {"status": "success", "tool_id": "create_four_color_plot", "time_seconds": 19.71760106086731, "inputs": {"plottitle": "FourColor", "height": "700", "width": "300", "xlabel": "Distance from motif midpoint (bp)", "input": {"src": "hda", "id": "2891970512fa2d5a"}, "dpi": "100"}, "tool_version": "1.0.0", "job": {"stdout": "\nprocessed sequences: ['>chr12:241974-242003(+)', '>chr7:649047-649076(+)', '>chr13:23851-23880(-)', '>chr6:23851-23880(-)']\n\nplotData : \n[[2 4 1 4 4 3 1 1 4 4 2 2 2 3 2 3 4 2 3 2 2 2 4 3 1 1 4 3 1]\n [2 2 4 3 4 2 1 2 2 4 3 4 1 3 1 3 4 3 3 3 2 2 4 4 1 1 1 1 3]\n [3 3 3 2 4 4 1 1 1 3 2 1 3 1 2 4 2 3 1 1 4 1 1 3 2 4 1 3 4]\n [4 1 2 2 2 4 4 1 2 3 4 1 1 3 2 4 2 3 1 2 3 1 4 3 3 2 2 3 2]]\n", "job_stderr": "", "create_time": "2019-08-14T18:51:27.444500", "id": "5729865256bc2525", "tool_stderr": "/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py:9: UserWarning: \nThis call to matplotlib.use() has no effect because the backend has already\nbeen chosen; matplotlib.use() must be called *before* pylab, matplotlib.pyplot,\nor matplotlib.backends is imported for the first time.\n\nThe backend was *originally* set to 'MacOSX' by the following code:\n  File \"/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py\", line 3, in <module>\n    import matplotlib.pyplot as plt\n  File \"/Users/pxk284/miniconda3/envs/mulled-v1-84070605f12759113e8b815a3a3d5b99995a2ac8863de1122d8cc0b269d84bdd/lib/python2.7/site-packages/matplotlib/pyplot.py\", line 71, in <module>\n    from matplotlib.backends import pylab_setup\n  File \"/Users/pxk284/miniconda3/envs/mulled-v1-84070605f12759113e8b815a3a3d5b99995a2ac8863de1122d8cc0b269d84bdd/lib/python2.7/site-packages/matplotlib/backends/__init__.py\", line 16, in <module>\n    line for line in traceback.format_stack()\n\n\n  matplotlib.use('Agg')\n", "command_line": "python '/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py' '/private/var/folders/zz/zyxvpxvq6csfxvn_n0030flm00r3mx/T/tmpes8zse/files/3/8/c/dataset_38c3f176-a10f-4c20-a058-dc6f4c09ea04.dat' '700' '300' '100' 'Distance from motif midpoint (bp)' 'FourColor' 'fourcolorplot.png'", "exit_code": 0, "state": "ok", "params": {"plottitle": "\"FourColor\"", "dbkey": "\"?\"", "height": "\"700\"", "width": "\"300\"", "xlabel": "\"Distance from motif midpoint (bp)\"", "chromInfo": "\"/private/var/folders/zz/zyxvpxvq6csfxvn_n0030flm00r3mx/T/tmpes8zse/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dpi": "\"100\""}, "tool_stdout": "\nprocessed sequences: ['>chr12:241974-242003(+)', '>chr7:649047-649076(+)', '>chr13:23851-23880(-)', '>chr6:23851-23880(-)']\n\nplotData : \n[[2 4 1 4 4 3 1 1 4 4 2 2 2 3 2 3 4 2 3 2 2 2 4 3 1 1 4 3 1]\n [2 2 4 3 4 2 1 2 2 4 3 4 1 3 1 3 4 3 3 3 2 2 4 4 1 1 1 1 3]\n [3 3 3 2 4 4 1 1 1 3 2 1 3 1 2 4 2 3 1 1 4 1 1 3 2 4 1 3 4]\n [4 1 2 2 2 4 4 1 2 3 4 1 1 3 2 4 2 3 1 2 3 1 4 3 3 2 2 3 2]]\n", "job_messages": [], "tool_id": "create_four_color_plot", "update_time": "2019-08-14T18:51:35.222408", "outputs": {"fourcolorplot_image": {"src": "hda", "id": "5729865256bc2525", "uuid": "d98b4a20-3c90-4062-bde5-739076c6f1ea"}}, "history_id": "2891970512fa2d5a", "job_metrics": [], "job_stdout": "", "inputs": {"input": {"src": "hda", "id": "2891970512fa2d5a", "uuid": "38c3f176-a10f-4c20-a058-dc6f4c09ea04"}}, "stderr": "/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py:9: UserWarning: \nThis call to matplotlib.use() has no effect because the backend has already\nbeen chosen; matplotlib.use() must be called *before* pylab, matplotlib.pyplot,\nor matplotlib.backends is imported for the first time.\n\nThe backend was *originally* set to 'MacOSX' by the following code:\n  File \"/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py\", line 3, in <module>\n    import matplotlib.pyplot as plt\n  File \"/Users/pxk284/miniconda3/envs/mulled-v1-84070605f12759113e8b815a3a3d5b99995a2ac8863de1122d8cc0b269d84bdd/lib/python2.7/site-packages/matplotlib/pyplot.py\", line 71, in <module>\n    from matplotlib.backends import pylab_setup\n  File \"/Users/pxk284/miniconda3/envs/mulled-v1-84070605f12759113e8b815a3a3d5b99995a2ac8863de1122d8cc0b269d84bdd/lib/python2.7/site-packages/matplotlib/backends/__init__.py\", line 16, in <module>\n    line for line in traceback.format_stack()\n\n\n  matplotlib.use('Agg')\n", "model_class": "Job", "external_id": "15786", "user_email": "test@bx.psu.edu"}, "test_index": 0}, "id": "functional.test_toolbox.TestForTool_create_four_color_plot.test_tool_000000", "has_data": true}, {"data": {"status": "success", "tool_id": "create_four_color_plot", "time_seconds": 13.807144165039062, "inputs": {"plottitle": "FourColor", "height": "700", "width": "300", "xlabel": "Distance from motif midpoint (bp)", "input": {"src": "hda", "id": "54f2a3a23292eb07"}, "dpi": "100"}, "tool_version": "1.0.0", "job": {"stdout": "\nprocessed sequences: ['>chr10:703356-703417(+)', '>chr11:108899-108960(+)', '>chr10:454180-454241(+)', '>chr11:431636-431697(+)', '>chr10:75641-75702(+)']\n\nplotData : \n[[1 1 1 2 1 1 4 3 3 1 1 1 1 4 3 1 4 1 3 2 2 4 1 1 3 1 1 1 1 1 1 1 3 1 1 4 3\n  4 1 3 3 1 1 3 1 1 1 1 1 3 3 1 1 3 4 1 1 1 1 2 2 1 2 1 1 1 1 3 1 2 1 2 1 3\n  2 4 3 4 1 1 3]\n [2 2 1 2 4 1 2 4 4 1 1 1 4 1 4 1 2 2 1 4 1 1 4 1 2 4 4 3 2 1 4 1 2 2 2 1 3\n  2 2 3 3 4 2 4 4 1 2 2 2 2 2 2 3 1 1 1 1 1 1 2 1 3 4 3 3 3 1 3 1 1 2 3 1 1\n  3 4 4 4 1 4 4]\n [1 4 4 3 1 1 1 3 4 1 2 4 3 2 2 3 1 2 1 1 4 4 2 2 3 3 4 4 1 1 2 3 3 1 3 4 4\n  1 1 1 4 1 3 2 2 3 3 2 3 3 2 1 4 1 1 3 1 1 3 4 3 1 2 2 4 1 2 3 3 2 1 1 4 1\n  1 1 4 1 2 2 2]\n [1 1 1 3 1 3 4 3 3 2 2 2 1 4 3 1 3 1 3 1 2 2 3 4 2 3 4 2 2 3 4 1 3 4 2 3 4\n  3 2 3 4 1 2 2 2 2 2 4 4 1 2 1 1 2 4 3 3 3 2 2 2 3 3 2 3 1 3 1 3 1 3 2 4 4\n  2 3 3 1 1 2 2]\n [1 3 3 4 2 4 4 3 3 4 2 4 1 1 1 4 2 1 4 4 1 4 4 1 4 4 1 4 3 4 3 3 4 1 3 3 4\n  4 3 1 3 3 4 1 1 1 4 1 3 2 1 4 3 4 4 2 2 3 3 4 3 2 3 1 2 1 2 2 2 4 2 1 2 3\n  2 4 3 3 1 3 1]]\n", "job_stderr": "", "create_time": "2019-08-14T18:51:54.916000", "id": "8155e4b4bf1581ff", "tool_stderr": "/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py:9: UserWarning: \nThis call to matplotlib.use() has no effect because the backend has already\nbeen chosen; matplotlib.use() must be called *before* pylab, matplotlib.pyplot,\nor matplotlib.backends is imported for the first time.\n\nThe backend was *originally* set to 'MacOSX' by the following code:\n  File \"/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py\", line 3, in <module>\n    import matplotlib.pyplot as plt\n  File \"/Users/pxk284/miniconda3/envs/mulled-v1-84070605f12759113e8b815a3a3d5b99995a2ac8863de1122d8cc0b269d84bdd/lib/python2.7/site-packages/matplotlib/pyplot.py\", line 71, in <module>\n    from matplotlib.backends import pylab_setup\n  File \"/Users/pxk284/miniconda3/envs/mulled-v1-84070605f12759113e8b815a3a3d5b99995a2ac8863de1122d8cc0b269d84bdd/lib/python2.7/site-packages/matplotlib/backends/__init__.py\", line 16, in <module>\n    line for line in traceback.format_stack()\n\n\n  matplotlib.use('Agg')\n", "command_line": "python '/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py' '/private/var/folders/zz/zyxvpxvq6csfxvn_n0030flm00r3mx/T/tmpes8zse/files/4/2/1/dataset_421e186d-234b-4d25-b892-f75029465590.dat' '700' '300' '100' 'Distance from motif midpoint (bp)' 'FourColor' 'fourcolorplot.png'", "exit_code": 0, "state": "ok", "params": {"plottitle": "\"FourColor\"", "dbkey": "\"?\"", "height": "\"700\"", "width": "\"300\"", "xlabel": "\"Distance from motif midpoint (bp)\"", "chromInfo": "\"/private/var/folders/zz/zyxvpxvq6csfxvn_n0030flm00r3mx/T/tmpes8zse/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dpi": "\"100\""}, "tool_stdout": "\nprocessed sequences: ['>chr10:703356-703417(+)', '>chr11:108899-108960(+)', '>chr10:454180-454241(+)', '>chr11:431636-431697(+)', '>chr10:75641-75702(+)']\n\nplotData : \n[[1 1 1 2 1 1 4 3 3 1 1 1 1 4 3 1 4 1 3 2 2 4 1 1 3 1 1 1 1 1 1 1 3 1 1 4 3\n  4 1 3 3 1 1 3 1 1 1 1 1 3 3 1 1 3 4 1 1 1 1 2 2 1 2 1 1 1 1 3 1 2 1 2 1 3\n  2 4 3 4 1 1 3]\n [2 2 1 2 4 1 2 4 4 1 1 1 4 1 4 1 2 2 1 4 1 1 4 1 2 4 4 3 2 1 4 1 2 2 2 1 3\n  2 2 3 3 4 2 4 4 1 2 2 2 2 2 2 3 1 1 1 1 1 1 2 1 3 4 3 3 3 1 3 1 1 2 3 1 1\n  3 4 4 4 1 4 4]\n [1 4 4 3 1 1 1 3 4 1 2 4 3 2 2 3 1 2 1 1 4 4 2 2 3 3 4 4 1 1 2 3 3 1 3 4 4\n  1 1 1 4 1 3 2 2 3 3 2 3 3 2 1 4 1 1 3 1 1 3 4 3 1 2 2 4 1 2 3 3 2 1 1 4 1\n  1 1 4 1 2 2 2]\n [1 1 1 3 1 3 4 3 3 2 2 2 1 4 3 1 3 1 3 1 2 2 3 4 2 3 4 2 2 3 4 1 3 4 2 3 4\n  3 2 3 4 1 2 2 2 2 2 4 4 1 2 1 1 2 4 3 3 3 2 2 2 3 3 2 3 1 3 1 3 1 3 2 4 4\n  2 3 3 1 1 2 2]\n [1 3 3 4 2 4 4 3 3 4 2 4 1 1 1 4 2 1 4 4 1 4 4 1 4 4 1 4 3 4 3 3 4 1 3 3 4\n  4 3 1 3 3 4 1 1 1 4 1 3 2 1 4 3 4 4 2 2 3 3 4 3 2 3 1 2 1 2 2 2 4 2 1 2 3\n  2 4 3 3 1 3 1]]\n", "job_messages": [], "tool_id": "create_four_color_plot", "update_time": "2019-08-14T18:52:01.027760", "outputs": {"fourcolorplot_image": {"src": "hda", "id": "8155e4b4bf1581ff", "uuid": "ebfd4953-4dd8-49d5-bf1c-85326c833999"}}, "history_id": "5729865256bc2525", "job_metrics": [], "job_stdout": "", "inputs": {"input": {"src": "hda", "id": "54f2a3a23292eb07", "uuid": "421e186d-234b-4d25-b892-f75029465590"}}, "stderr": "/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py:9: UserWarning: \nThis call to matplotlib.use() has no effect because the backend has already\nbeen chosen; matplotlib.use() must be called *before* pylab, matplotlib.pyplot,\nor matplotlib.backends is imported for the first time.\n\nThe backend was *originally* set to 'MacOSX' by the following code:\n  File \"/Users/pxk284/Desktop/IUC/ToolsToPublish/create_fourcolor_plot/create_fourcolor_plot.py\", line 3, in <module>\n    import matplotlib.pyplot as plt\n  File \"/Users/pxk284/miniconda3/envs/mulled-v1-84070605f12759113e8b815a3a3d5b99995a2ac8863de1122d8cc0b269d84bdd/lib/python2.7/site-packages/matplotlib/pyplot.py\", line 71, in <module>\n    from matplotlib.backends import pylab_setup\n  File \"/Users/pxk284/miniconda3/envs/mulled-v1-84070605f12759113e8b815a3a3d5b99995a2ac8863de1122d8cc0b269d84bdd/lib/python2.7/site-packages/matplotlib/backends/__init__.py\", line 16, in <module>\n    line for line in traceback.format_stack()\n\n\n  matplotlib.use('Agg')\n", "model_class": "Job", "external_id": "15939", "user_email": "test@bx.psu.edu"}, "test_index": 1}, "id": "functional.test_toolbox.TestForTool_create_four_color_plot.test_tool_000001", "has_data": true}], "version": "0.1", "exit_code": 0, "summary": {"num_skips": 0, "num_errors": 0, "num_failures": 0, "num_tests": 2}}